src/content/boundary/product-landing.json

```diff
@@ -10,7 +10,10 @@
 			"text": "Learn more about Boundary features",
 			"url": "/boundary/docs/overview/what-is-boundary"
 		},
-		"image": "https://www.datocms-assets.com/2885/1662053765-dev-dot-product-landing-what-is-boundary-diagram.png"
+		"image": {
+			"light": "https://www.datocms-assets.com/2885/1679095102-devdot-boundary_lm.png",
```

Comment:
We're updating both light and dark here because some of the assets were inconsistent.